ImperatorToCK3/Data_Files/configurables/religion_map.txt

```diff
@@ -11,29 +11,59 @@
+link = { ck3 = slavic_pagan imp = matrist_religion }
+
+# Steppe Religion - Melieism in the East, Tengri in the West
+link = { ck3 = tengri_pagan imp = eastern_animism impRegion = sarmatia_europea_region impRegion = vistulia_region impRegion = venedia_region impRegion = scandinavia_region impRegion = dacia_region impRegion = pannonia_region impRegion = moesia_region impRegion = thrace_region impRegion = macedonia_region impRegion = greece_region impRegion = illyria_region impRegion = germania_region impRegion = bohemia_area }
+link = { ck3 = melieism imp = eastern_animism }
+
+# Celtic
+link = { ck3 = briton_paganism imp = druidism impRegion = britain_region impRegion = scandinavia_region }
+link = { ck3 = gaelic_paganism imp = druidism impRegion = caledonia_region }
+link = { ck3 = gallic_celtic imp = druidism impRegion = aquitaine_region impRegion = transalpine_gaul_region impRegion = armorica_region impRegion = central_gaul_region impRegion = rhaetia_region impRegion = cisalpine_gaul_region impRegion = central_italy_region impRegion = magna_graecia_region impRegion = belgica_region impRegion = germania_superior_region }
+link = { ck3 = celtiberian_paganism imp = druidism impRegion = tarraconensis_region impRegion = gallaecia_region impRegion = lusitania_region impRegion = contestania_region impRegion = baetica_region }
+link = { ck3 = celtic_pagan imp = druidism } # Fallback
+
 #Jewish
-link = { ck3 = haymanot imp = judaism ck3Province = 8312 ck3Province = 8303 ck3Province = 8366 ck3Province = 8371 }
+link = { ck3 = haymanot imp = judaism impRegion = punt_region }
+link = { ck3 = kabarism imp = judaism impRegion = sarmatia_asiatica_region impRegion = taurica_region impRegion = colchis_region impRegion = bithynia_region impRegion = cappadocia_region impRegion = galatia_region impRegion = cilicia_region }
+link = { ck3 = karaism imp = judaism impRegion = assyria_region impRegion = mesopotamia_region }
+link = { ck3 = malabarism imp = judaism impRegion = gandhara_region impRegion = maru_region impRegion = avanti_region impRegion = madhyadesa_region impRegion = pracya_region impRegion = vindhyaprstha_region impRegion = dravida_region impRegion = aparanta_region impRegion = karnata_region }
+link = { ck3 = samaritan imp = judaism ck3Province = 5961 } # Samaria
 link = { ck3 = rabbinism imp = judaism }
 
+# Germanic
+link = { ck3 = finnish_pagan imp = germanic_religion impProvince = 5976 }
```

Comment:
Why not just map to Germanic? AFAIK Finnish and Germanic paganisms weren't very related.

Reply:
ImpProvince 5976 is a wasteland that maps onto CK3 Finland. We do this to avoid Christian realms there, but it would be unrealistic to image a Norse conquest already. Same logic in the equivalent culture mapping link.